src/comp_loinc/groups/groups_builders.py

```diff
@@ -446,6 +446,9 @@ def do_index_parts_tree(self):
                     case LoincTreeEdges.tree_parent:
                         child_part_node.parents[parent_part_number] = parent_part_node
                         parent_part_node.children[part_number] = child_part_node
+                    case LoincDanglingNlpEdges.nlp_parent:
```

Comment:
Added to `group_builders.py`

I added it to `group_builders_wip.py` too. It's here alongside `case`s for `LoincTreeEdges.tree_parent` and `LoincPartEdge.parent_comp_by_system`.

I only did a test run of `lp-hierarchy-all` and examined `loinc-part-hierarchy-all.owl`. And ran a full `--fast-run` build via the PR QC.

But I haven't ran / examined how the NLP dangling parts show up in grouping classes.